include/onnxruntime/core/session/onnxruntime_session_options_config_keys.h

```diff
@@ -52,6 +52,8 @@ static const char* const kOrtSessionOptionsEnableQuantQDQ = "session.enable_quan
 static const char* const kOrtSessionOptionsEnableGeluApproximation = "optimization.enable_gelu_approximation";
 
 // Enable or disable using device allocator for allocating initialized tensor memory. "1": enable; "0": disable. The default is "0".
+// Using device allocators means the memory allocation is made "directly" using the appropriate system call
```

Comment:
nit: Since this file is public facing, I suppose we can replace 'directly' with 'malloc/new'.

Reply:
sure